packages/front-end/components/Settings/SubscriptionInfo.tsx

```diff
@@ -26,6 +37,34 @@ export default function SubscriptionInfo() {
           commercialFeature={null}
         />
       )}
+      {cancelSubscriptionModal && (
+        <Modal
+          open={true}
+          header="Are you sure you want to cancel?"
+          trackingEventModalType="cancel-subscription"
+          close={() => setCancelSubscriptionModal(false)}
+          cta="Yes, Cancel Subscription"
+          closeCta="Keep Subscription"
+          submitColor="danger"
+          submit={async () => {
+            await apiCall("/subscription/cancel", { method: "POST" });
+            refreshOrganization();
+          }}
+        >
+          <>
+            <p>
+              If you cancel, you will continue to have access to your
+              <strong> {planNameFromAccountPlan(accountPlan)} Plan </strong>
+              features until your current billing period ends on{" "}
+              {subscription?.nextBillDate}.
+            </p>
+            <Callout status="warning">
+              You account can still acrue CDN usage charges. If you&apos;d like
+              to prevent that, you can disable live feature flags.
```

Comment:
by "disable live feature flags" do you mean "remove Growthbook SDK from your code base"?  I find the phrasing you have here a bit ambiguous and sounding like they can flip some switch within Growthbook.